drivers/clock_control/stm32_ll_clock.c

```diff
@@ -375,6 +423,24 @@ static int stm32_clock_control_init(struct device *dev)
 
 #endif /* CONFIG_CLOCK_STM32_SYSCLK_SRC_... */
 
+#ifndef CONFIG_CLOCK_STM32_MCO1_SRC_NOCLOCK
```

Comment:
Please put this under a specific function (stm32clock_control_mco_init for instance)
stm32_clock_control_init is already rather long, we should not add too much things.

Reply:
Besides, according to Reference manual, it should be done before PLL activation.